src/StreamJsonRpc.Shared/JsonRpc.cs

```diff
@@ -145,7 +145,7 @@ public JsonRpc(Stream sendingStream, Stream receivingStream, object target = nul
         public JsonRpc(DelimitedMessageHandler messageHandler, object target = null)
         {
             Requires.NotNull(messageHandler, nameof(messageHandler));
-
+            
```

Comment:
@AArnott Not sure what's going on, but I just can't see what whitespace you're referring to?